src/client/app/containers/editor.ts

```diff
@@ -39,6 +39,16 @@ export class Editor implements AfterViewInit {
      */
     async ngAfterViewInit() {
         this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let editor = this._monacoEditor;
```

Comment:
Why are you creating a new variable for this?  why not just re-use `this._monacoEditor` as is, and call `addAction` on it.